### ISSUE TITLE
Issue with interpolation method linearNDFast

### ISSUE THREAD

**Issue by trondkr** [OpenDrift/opendrift#51]
I experience a problem using the new interpolation method linearNDFast which I cant explain. I am using two ROMS (Regional Ocean Modeling System) as forcing and I do **linearNDFast** interpolation for the particles. I have run this setup successfully using **linearND** and **nearest** as interpolation with success, but it fails with the error pasted below using **linearNDFast**. It seems x and y water velocities suddenly are inf in some grid cells  (y_sea_water_velocity: -**inf** (min) 0.420177 (max)), probably caused by the interpolation. If you could solve this issue that would be great. Thanks.

```
DEBUG: Unstaggering y_sea_water_velocity
DEBUG: Fetched env-block (size 340x351x12) for time before (2016-05-07 01:00:00)
DEBUG: Interpolating before (2016-05-07 01:00:00) in space  (linearNDFast)
DEBUG: Initialising interpolator.
DEBUG: No time interpolation needed - right on time.
DEBUG: Rotating vectors between 0.0 and 0.0 degrees.
DEBUG: Masking 5 elements outside coverage
DEBUG: Data missing for 7 elements.
DEBUG: Calling reader roms native_0
DEBUG: ----------------------------------------
DEBUG: Data needed for 7 elements
DEBUG: Reader time:
		2016-05-06 12:00:00 (before)
		2016-05-07 12:00:00 (after)
DEBUG: Interpolating before (2016-05-06 12:00:00) in space  (linearNDFast)
DEBUG: Initialising interpolator.
DEBUG: Interpolating after (2016-05-07 12:00:00) in space  (linearNDFast)
DEBUG: Initialising interpolator.
DEBUG: Interpolating before (2016-05-06 12:00:00, weight 0.46) and
		      after (2016-05-07 12:00:00, weight 0.54) in time
DEBUG: Rotating vectors between 0.0 and 0.0 degrees.
DEBUG: Obtained data for all elements.
DEBUG: ---------------------------------------
DEBUG: Finished processing all variable groups
DEBUG: ------------ SUMMARY -------------
DEBUG:     x_sea_water_velocity: -inf (min) 0.305204 (max)
DEBUG:     y_sea_water_velocity: -inf (min) 0.420177 (max)
DEBUG: ---------------------------------
DEBUG: 		124 active elements
DEBUG: 		69.6269992313 <- latitude  -> 70.1287789924
DEBUG: 		17.1623669881 <- longitude -> 18.7455382663
DEBUG: 		-157.5   <- z ->   -0.5
DEBUG: ---------------------------------
INFO: ========================
INFO: End of simulation:
INFO: undefined inverse geodesic (may be an antipodal point)
INFO: Traceback (most recent call last):
  File "/work-common/shared/nn9297k/Kelp/opendrift/opendrift/models/basemodel.py", line 1641, in run
    self.update()
  File "/work-common/shared/nn9297k/Kelp/kelp/kelpClass.py", line 195, in update
    self.advect_ocean_current()
  File "/work-common/shared/nn9297k/Kelp/opendrift/opendrift/models/physics_methods.py", line 110, in advect_ocean_current
    mid_env['y_sea_water_velocity'])
  File "/work-common/shared/nn9297k/Kelp/opendrift/opendrift/models/basemodel.py", line 2626, in update_positions
    azimuth, velocity*self.time_step.total_seconds())
  File "/work/apps/python/2.7.9-dso-gnu/lib/python2.7/site-packages/pyproj/__init__.py", line 780, in fwd
    _proj.Geod._fwd(self, inx, iny, inz, ind, radians=radians)
  File "_proj.pyx", line 427, in _proj.Geod._fwd (_proj.c:4299)
ValueError: undefined inverse geodesic (may be an antipodal point)
```

**Comment by knutfrode** [comment]
This appears to be the same problem that @kristokv also experienced recently.
It seems that once in a while, extreme values are returned from the linearNDFast interpolation.
A recent commit could potentially solve this, though not too likely. Did you see an improvement, Kristina?
And: do you know if this happens e.g. close to land, or close to model domain boundary?

I might need help or hint from @babrodtk to solve this problem.

**Comment by knutfrode** [comment]
Fixed in d2b839d6c5dbccce32ee9c84b4789041ad8f5a70
